Objects/Objects/BuiltElements/Revit/FamilyInstance.cs

```diff
@@ -1,60 +1,80 @@
-﻿using System;
-using Objects.Geometry;
+﻿using Objects.Geometry;
 using Objects.Utils;
 using Speckle.Core.Kits;
 using Speckle.Core.Models;
+using Speckle.Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using Speckle.Newtonsoft.Json;
 
 namespace Objects.BuiltElements.Revit
 {
-  public class FamilyInstance : Base, IDisplayMesh, IDisplayValue<List<Mesh>>
-  {
-    public Point basePoint { get; set; }
-    public string family { get; set; }
-    public string type { get; set; }
-    public string category { get; set; }
-    public Level level { get; set; }
-    public double rotation { get; set; }
-    public bool facingFlipped { get; set; }
-    public bool handFlipped { get; set; }
-    public bool mirrored { get; set; }
-    public Base parameters { get; set; }
-    public string elementId { get; set; }
-
-    [DetachProperty]
-    public List<Base> elements { get; set; }
-    
-    [DetachProperty]
-    public List<Mesh> displayValue { get; set; }
-
-    public string units { get; set; }
-
-    public FamilyInstance() { }
-
-    [SchemaInfo("FamilyInstance", "Creates a Revit family instance", "Revit", "Families")]
-    public FamilyInstance(Point basePoint, string family, string type, Level level,
-      double rotation = 0, bool facingFlipped = false, bool handFlipped = false,
-      List<Parameter> parameters = null)
+    public class FamilyInstance : Base, IDisplayMesh, IDisplayValue<List<Mesh>>, IMEPElement
     {
-      this.basePoint = basePoint;
-      this.family = family;
-      this.type = type;
-      this.level = level;
-      this.rotation = rotation;
-      this.facingFlipped = facingFlipped;
-      this.handFlipped = handFlipped;
-      this.mirrored = false;
-      this.parameters = parameters.ToBase();
-    }
-    
-    #region Obsolete Members
-    [JsonIgnore, Obsolete("Use " + nameof(displayValue) + " instead")]
-    public Mesh displayMesh {
-      get => displayValue?.FirstOrDefault();
-      set => displayValue = new List<Mesh> {value};
+        public Point basePoint { get; set; }
+        public string family { get; set; }
+        public string type { get; set; }
+        public string category { get; set; }
+        public Level level { get; set; }
+        public double rotation { get; set; }
+        public bool facingFlipped { get; set; }
+        public bool handFlipped { get; set; }
+        public bool mirrored { get; set; }
+        public Base parameters { get; set; }
+        public string elementId { get; set; }
+        public Connector[] connectors { get; set; }
```

Comment:
So instead of adding these properties to FamilyInstance, Let's make a new class that inherits from it!